backend/app/models/__init__.py

```
@@ -62,3 +62,18 @@
     OpenAIConversationBase,
     OpenAIConversationCreate,
 )
+
+from .prompt import (
+    Prompt,
```

Ruff (0.12.2)

`.prompt.Prompt` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)